src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -24,7 +26,6 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "ComputerInfo",
         HelpUri = "https://go.microsoft.com/fwlink/?LinkId=799466")]
-    [Alias("gin")]
```

Comment:
For those aliases that are already declared in `InitialSessionState.cs`, I prefer to keep them there for now, for the sake of backward compatibility.
For those aliases that are declared with `AliasAttribute`, I also prefer to keep them that way unless it's a core cmdlet that should be set with options other than 'None'.